src/OnboardingSPA/pages/Steps/DesignHomepageMenu/index.js

```diff
@@ -16,6 +16,7 @@ import { LivePreviewSelectableCard } from '../../../components/LivePreview';
 import HeadingWithSubHeading from '../../../components/HeadingWithSubHeading';
```

Comment:
![Screenshot 2022-11-17 at 12 21 55 PM](https://user-images.githubusercontent.com/38878906/202376786-5e6b0953-0948-45d4-b7d0-4ee019b10913.png)
Homepage is empty please re-check if you missed anything when bringing in changes from my PR

Reply:
Haha sorry in front end i did not add 'yith-wonder' to slug names as per new styles